bundle/config/mutator/process_target_mode.go

```diff
@@ -105,6 +105,12 @@ func transformDevelopmentMode(ctx context.Context, b *bundle.Bundle) diag.Diagno
 		// (registered models in Unity Catalog don't yet support tags)
 	}
 
+	for i := range r.Schemas {
+		prefix = "dev_" + b.Config.Workspace.CurrentUser.ShortName + "_"
```

Comment:
Schema prefixing like this for development mode seems reasonable. But might be up for devbate and it's certainly not something all customers will want.

Once we add some configuration for `mode: development` we should likely make data prefixing separately configurable.

Reply:
Yeah agreed. We can keep the prefixing as the default behaviour though to keep it consistent with the other resources.